recipes/flowtorch/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "flowtorch" %}
+{% set version = "0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 775e69535dc54615a6c92d835e29fadc8d2bcc79ba13ce3eca06ee13ec9d02a5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:    
+    - pip    
+    - setuptools-scm
+    - python >=3.7
+    - pytorch >=1.8.1
```

Comment:
It doesn't seem like `pytorch` is needed to build the package, only to import/run it.
```suggestion
```

Reply:
Ok. Let me see what happens if I remove it.

Reply:
Not generally, no. But you have it listed under `run`, which *IS* installed for the tests, since everything under `run` is installed whenever you install the package.

Reply:
Thank you. Done.